tests/validation/requirements_v3api.txt

```diff
@@ -18,4 +18,4 @@ python-digitalocean==1.13.2
 PyYAML==5.2
 requests==2.22.0
 rsa==4.0
-websocket-client==0.53.0
+websocket-client==0.57.0
```

Comment:
We sure no other test was impacted by bumping this version?

Reply:
I did not see any other places under `v3_api` except this PR that uses this library